lib/chip-types/resource-type-field.js

```diff
@@ -1,16 +1,21 @@
 var Promise = require("bluebird");
 
-function ResourceTypeField(declaration){
-	this.name = declaration.name;
-	this.type_name = declaration.type;
-	this.human_readable_name = declaration.human_readable_name || null;
-	var type_constructor = Sealious.ChipManager.get_chip("field_type", declaration.type);
-	this.type_parameters = declaration.params;
-	this.type = new type_constructor();
-	this.type.set_params(declaration.params);
-	this.required = declaration.required || false;
-	this.derived = declaration.derived || false;
-}
+function ResourceTypeField(declaration) {
+    this.name = declaration.name;
+    this.type_name = declaration.type;
+    this.human_readable_name = declaration.human_readable_name || null;
+    var type_constructor = Sealious.ChipManager.get_chip("field_type", declaration.type);
+    this.type_parameters = declaration.params;
+    //var check_field_type = Sealious.ChipManager.chip_exists("field_type", this.type_name);
+    try {
+        this.type = new type_constructor();
+        this.type.set_params(declaration.params);
+        this.required = declaration.required || false;
+        this.derived = declaration.derived || false;
+    } catch (e) {
+        throw new Sealious.Errors.DeveloperError("Not exist field_type." + this.type_name);
+    }
```

Comment:
1. Powinno być bez try_catch. Można to zrobić zwykłym `if`-em, korzystając z `ChipManager.chip_exists`
2. Treść błędu należy ugramatycznić: "Field type `typ_pola` was not found.".
3. Trzeba by było pomyśleć nad dodaniem do treści błędu informacji o tym, w jakim typie zasobu pojawiła się ta niepoprawna deklaracja pola. Do tego struktura samego ResourceType i ResourceFieldType musi ulec małej zmianie - do funkcji tworzącej ResourceTypeField należy dodać drugi argument, który jest odnośnikiem do obiektu typu pola, który ją wywołuje